src/js/components/LandingSection.tsx

```diff
@@ -27,73 +27,48 @@ const springPageDown = (setY: Function): void => {
 const LandingSection = ({setY}: {setY: Function}): JSX.Element => {
 	//const [, setY] = useSpring(() => ({ y: 0 }));
 
-	const shareBtn = <ShareButton
-		absolute={!isPortraitMobile()}
-		className="btn-transparent fill"
-		style={isPortraitMobile() ? null : {bottom: 30, left: 30}}
-		title={"My-Loop"}
-		image={"https://my.good-loop.com/img/GoodLoopLogos_Good-Loop_AltLogo_Colour.png"}
-		description={"Using ads for good"}
-		url={window.location.href}>
-			Share
-	</ShareButton>;
+	const scrolltoNewsletter = () => {
+		document.getElementById("subscription-box")!.scrollIntoView({behavior: "smooth"});
```

Comment:
What does `!.` do? I haven't met that before? (and I can't google it!)

Reply:
Typescript will have an error `Type error: Object is possibly 'null'.  TS2531` to inform me this could be null. The `!` operator is telling typescript that the `document.ElementById("subscription-box")` does exist, removing the warning. 
The code ran fine either with or without the `!` operator, should I leave the error message on (i.e. do not add the `!`) instead?

Reply:
You can leave it as is.

We'll probably switch this file from .tsx to .jsx at some point.

NB: I'd quite like to use typescript. But when we tried it before, we ran into a set of build problems. So we're sticking with js for now.

Reply:
PS - Thanks for the explanation! I have learned something about .ts